packages/server/src/api/routes/tests/automation.spec.ts

```diff
@@ -436,4 +443,36 @@ describe("/automations", () => {
       expect(res).toEqual(true)
     })
   })
+
+  describe("Update Row Old / New Row comparison", () => {
+    it.each([
+      { oldCity: "asdsadsadsad", newCity: "new", expectedResult: false },
+      { oldCity: "Belfast", newCity: "Belfast", expectedResult: true },
```

Comment:
nit: you could very easily derive the `expectedResult` by doing `oldCity === newCity`. If we ever expanded these tests it would make them a fair bit easier to write, removing the scope of potential human error.